### ISSUE TITLE
file_get_contents(): Passing null to parameter #2 ($use_include_path) of type bool is deprecated 8192

### ISSUE THREAD

**Issue by thetestcoder** [dompdf/dompdf#2732]
Getting this issue file_get_contents(): Passing null to parameter #2 ($use_include_path) of type bool is deprecated 8192.

#1 E:\\xampp\\htdocs\\bipworks\\vendor\\dompdf\\dompdf\\src\\Helpers.php(839): file_get_contents('E:\\\\xampp\\\\htdocs...', NULL, NULL, 0)
#2 E:\\xampp\\htdocs\\bipworks\\vendor\\dompdf\\dompdf\\src\\Css\\Stylesheet.php(367): Dompdf\\Helpers::getFileContent('E:\\\\xampp\\\\htdocs...', NULL)
#3 E:\\xampp\\htdocs\\bipworks\\vendor\\dompdf\\dompdf\\src\\Dompdf.php(572): Dompdf\\Css\\Stylesheet->load_css_file('E:\\\\xampp\\\\htdocs...', 1)
#4 E:\\xampp\\htdocs\\bipworks\\vendor\\dompdf\\dompdf\\src\\Dompdf.php(724): Dompdf\\Dompdf->processHtml()

**Comment by Mellthas** [comment]
This is already fixed for the next release, see #2659.

**Comment by tiagofrancafernandes** [comment]
If use  `barryvdh/laravel-dompdf`, se to `^1.0.0` version
If use `dompdf/dompdf` set to `^1.2.2`

on composer.json:

```
"barryvdh/laravel-dompdf": "^1.0.0"
```

Or 
```
composer require barryvdh/laravel-dompdf:"^1.0.0"
```
